salt/_states/metalk8s.py

```diff
@@ -87,8 +87,10 @@ def module_run(name, attemps=1, sleep_time=10, **kwargs):
                 **kwargs
             )
         except Exception as exc:  # pylint: disable=broad-except
-            ret['comment'] = str(exc)
-
+            if name == str('metalk8s_etcd.check_etcd_health'):
+                ret['comment'] = None
+            else:
+                ret['comment'] = str(exc)
```

Comment:
Nop it's not here.
You need to change in the module itself the log.warning.
But we may want to have this log somehow maybe we should log it in debug and raise it so we will have it in the resulting output and when we retry it's only printed in debug mode

Reply:
@TeddyAndrieux Since we own the `metalk8s_etcd` module, why not give a `retry` argument directly to the `check_etcd_health` method?

Reply:
Also, that's not the only retry that will print error logs... Either each `retry` we need will print logs unless we catch it here (looks like a hack to me), or we find some way to catch logs while retrying (which we may not want to do, since logs are useful).

Reply:
We can do it directly in `check_etcd_health` but it's kind of strange to me that a module automatically retry. It's more something we want to do in a state not when calling a module...
But I think in the module in case of critical error we should only raise this error and not log anything and then it's the state that handle this, so in our case we controle the retrying state so when retrying we just ignore (or maybe log in debug to say "attemps x: got this error: blabla")

Reply:
And yes it's not the only retry we will have but I think we should just not log in error in our modules just raise in case of error and let the caller handle this